crates/node/net/src/driver.rs

```diff
@@ -146,13 +154,21 @@ impl Network {
                             warn!(err = ?send_err, "Impossible to send a request to the discovery handler. The channel connection is dropped.");
                         }
                     },
-                    req = rpc.recv() => {
+                    req = p2p_rpc.recv() => {
                         let Some(req) = req else {
                             trace!("Receiver `None` rpc request");
                             continue;
                         };
                         req.handle(&self.gossip, &handler);
                     },
+                    req = rollup_rpc.recv() => {
+                        let Some(req) = req else {
+                            trace!("Receiver `None` rpc request");
+                            continue;
+                        };
+                        // TODO(@theochap): add engine driver here.
```

Comment:
How is the engine driver going to slot in here? The `kona-node-service` crate is what orchestrates all the different actors (and their drivers). If we pull the engine driver into the `NetworkDriver` here, how do we access it from inside `kona-node-service`?